README.md

```diff
@@ -1 +1,173 @@
-A React SPA with Space X API
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Yes! Great suggestion!!! Thank you very much for your support and feedback! 💯